<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gameStartWindow</class>
 <widget class="QMainWindow" name="gameStartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="exLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>오뮤 다예쁨체</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>다가오는 장애물들을 피해 

최대한 많은 구슬을 모아보자!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
    <property name="indent">
     <number>6</number>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>오뮤 다예쁨체</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(100, 229, 220);</string>
    </property>
    <property name="text">
     <string>게임시작</string>
    </property>
   </widget>
   <widget class="QLabel" name="ex2Label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>오뮤 다예쁨체</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>기록을 남기고 싶다면?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>오뮤 다예쁨체</family>
      <pointsize>11</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 4px;
	background-color: rgb(255, 253, 180);
}</string>
    </property>
    <property name="text">
     <string>로그인</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signupButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>오뮤 다예쁨체</family>
      <pointsize>11</pointsize>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 4px;
	background-color: rgb(250, 224, 255);
}</string>
    </property>
    <property name="text">
     <string>회원가입</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>538</width>
      <height>376</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/sky.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="howtoButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>여기어때 잘난체 2</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border-radius: 40px;
	color: rgb(255, 85, 127);
	background-color: rgb(255, 253, 180, 0
);
}</string>
    </property>
    <property name="text">
     <string>게임방법</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>exLabel</zorder>
   <zorder>startButton</zorder>
   <zorder>ex2Label</zorder>
   <zorder>loginButton</zorder>
   <zorder>signupButton</zorder>
   <zorder>howtoButton</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
